data/jobs.txt

```diff
@@ -1571,6 +1576,136 @@ mission "Cargo [4]"
+	name "Mining for <planet>"
+	description "Mine 10 tons of iron from asteroids and deliver it to the factories on <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 40
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 4
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes factory
+	on complete
+		outfit Iron -10
+		payment 16000
+		dialog "You drop off your cargo and collect your payment of <payment>."
+
+mission "Small Mining [1]"
+	name "Mining for <planet>"
+	description "Mine 10 tons of aluminum from asteroids and deliver it to the shipyard on <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 35
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 3
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes shipyard
+	on complete
+		outfit Aluminum -10
+		payment 22000
+		dialog "You drop off your cargo and collect your payment of <payment>."
+
+mission "Small Mining [2]"
+	name "Mining for <planet>"
+	description "Mine 10 tons of titanium from asteroids and deliver it to the shipyard on <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 30
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 3
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes shipyard
+	on complete
+		outfit Titanium -10
+		payment 29000
+		dialog "You drop off your cargo and collect your payment of <payment>."
+
+mission "Small Mining [3]"
+	name "Mining for <planet>"
+	description "Mine 10 tons of copper from asteroids and deliver it to <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 25
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 4
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes shipyard factory
+	on complete
+		outfit Copper -10
+		payment 34000
+		dialog "You drop off your cargo and collect your payment of <payment>."
+
+mission "Small Mining [4]"
+	name "Mining for <planet>"
+	description "Mine 10 tons of neodynium from asteroids and deliver it to the shipyard on <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 20
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 3
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes shipyard
+	on complete
+		outfit Neodynium -10
+		payment 42000
+		dialog "You drop off your cargo and collect your payment of <payment>."
+
+mission "Small Mining [5]"
+	name "Mining for <planet>"
+	description "Mine 5 tons of tungsten from asteroids and deliver it to the shipyard on <destination>. Payment is <payment>."
+	job
+	repeat
+	to offer
+		random < 15
+	source
+		government Republic "Free Worlds" Syndicate Neutral
+	destination
+		distance 0 3
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes syMajor
```

Comment:
It was my intention to deliver tungsten only to major shipyards. I know that this way the mission cant be offered in every place, which is intentional.